### PR TITLE
feat: add support for value and orWhere methods on Eloquent Builder

### PR DESCRIPTION
**Pull request by robbplo**
- [x] Added or updated tests
- [ ] Documented user facing changes
- [x] Updated CHANGELOG.md

**Changes**

Adds support for value and orWhere methods on the Eloquent query builder

### CODE REVIEW

File: tests/Rules/Data/model-property-builder.php [hunk 1]
```diff
@@ -22,6 +22,8 @@ function bar(\Illuminate\Database\Eloquent\Builder $builder): \Illuminate\Databa
 
 \App\User::query()->whereColumn('foo', '=', 'foo');
 \App\User::query()->where(getKey(), '=', 'foo');
+\App\User::query()->orWhere('foo', '=', 'foo');
```
Comment (szepeviktor) [inline tests/Rules/Data/model-property-builder.php:25]: Could you add this?

```suggestion
\App\User::query()->orWhere('foo', '=', 'foo');
\App\User::query()->orWhere('foo', 'foo');
```

Comment (szepeviktor) [reply tests/Rules/Data/model-property-builder.php:25]: Thank you.

File: tests/Rules/Data/model-property-builder.php [hunk 2]
```diff
@@ -22,6 +22,9 @@ function bar(\Illuminate\Database\Eloquent\Builder $builder): \Illuminate\Databa
...
+\App\User::query()->orWhere('foo', 'foo');
```
Comment (canvural) [inline tests/Rules/Data/model-property-builder.php:26]: `orWhere` also accepts arrays. With the array key being the model property. So can you add a test case like
```php
\App\User::query()->orWhere([
	'foo' => 'bar'
]);
```

**Comment by canvural** [comment]
All looks good! Thank you!

**Pull request merged**